### DISCUSSION TITLE
Getting all current running processes?

### DISCUSSION THREAD

**Discussion by Smotto** [halildurmus/win32#617] [Q&A] [answered]
**How would I go about grabbing all running processes?** 

I know .net has a Process class [.net](https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.process?view=net-7.0)

And win32 has something maybe I'm going for? [win32](https://learn.microsoft.com/en-us/windows/win32/psapi/enumerating-all-processes)

Could I wrap this function? [nf-psapi-enumprocesses](https://learn.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-enumprocesses)

I'm new and not sure if it'd be possible. I'd like a point in the right direction :)
Edit: I went through the examples and I found the window.dart and modules.dart. I think those are what I needed.

**Comment by Far-Se** [comment, marked as answer]
Here is an example how to use EnumProcesses:
https://github.com/Far-Se/tabame/blob/eb8d2804db3785b2d7feb754de2c524ce6b67c8c/lib/models/win32/win32.dart#L110
